client/src/featureform/metadata_repository.py

```diff
@@ -257,6 +259,12 @@ def get_labels(self) -> List[Label]:
 
     def get_source_variant(self, name: str, variant: str) -> SourceVariant:
         result = self.db.get_source_variant(name, variant)
+        name_variant_list = []
+        input_json = json.loads(result["inputs"] if result["inputs"] else [])
+        if input_json and len(input_json[0]):  # we store empty inputs as [[]]
+            name_variant_list = [
+                pb.NameVariant(name=v[0], variant=v[1]) for v in input_json
```

Comment:
Does this need to be a proto?

Reply:
it doesn't have to, it can be an inline dictionary. I'll change it in a bit.

Reply:
done.